packages/victory-core/src/victory-util/with-default-props.ts

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+/**
+ * Same definition as `React.FunctionComponent<TProps>`,
+ * except stronger types for `defaultProps`.
+ */
+export type FCWithDefaultProps<TProps, TDefaultProps> = {
+  (props: TProps): React.ReactElement | null;
+  defaultProps: TDefaultProps;
+} & Pick<React.FC<TProps>, "propTypes" | "contextTypes" | "displayName">;
+
+/**
+ * Provides the maximum type-safety for the `Component.defaultProps` property.
+ */
+export function withDefaultProps<
```

Comment:
Ignore this whole file.  It was an experiment to improve TS types.
It was successful, but, I later found a better way to do this, and included that solution in my "infrastructure" PR.

Reply:
Removed.